notifications/rest/users/create-user/create-user.3.x.js

```diff
@@ -0,0 +1,16 @@
+// Download the Node helper library from www.twilio.com/docs/libraries/node#installation
+// These identifiers are your accountSid and authToken from
+// https://www.twilio.com/console
+const accountSid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX';
+const authToken = 'your_auth_token';
+const client = require('twilio')(accountSid, authToken);
+
+client.notify
+  .services('ISXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX')
+  .users.create({
+    identity: 'User0001',
+    segments: ['premium', 'fitness-lifestyle']
+  })
+  .then(user => console.log(user.sid))
+  .catch(error => console.log(error))
```

Comment:
I wonder if it might be a good idea to not catch the error and just let the snippet tester handle it? wdyt @well1791 ?